synthetics-stats/src/entities/distributions.ts

```diff
@@ -0,0 +1,52 @@
+import { BigInt } from "@graphprotocol/graph-ts"
+import { EventData } from "../../../gmx-referrals/src/utils/eventData";
+import { Distribution, TokenPrice } from "../../generated/schema";
+
+export function saveDistribution(
+    receiver: string,
+    token: string,
+    amount: BigInt,
+    typeId: i32,
+    txHash: string,
+    blockNumber: i32,
+    timestamp: i32
+): void {
+    let id = receiver + ":" + txHash + ":" + typeId.toString();
```

Comment:
from first glance it's not obvious that "receiver" is actually same thing as "account". Maybe use account instead? I'm personally not sure